Improve shard allocation awareness documentation - part 2 (#8339)
opensearch-project/documentation-website: _tuning-your-cluster/index.md

### Shard allocation awareness

> You can set custom node attributes on OpenSearch nodes to be used for shard allocation awareness. For example, you can set the `zone` attribute on each node to represent the zone in which the node is located. You can also use the `zone` attribute to ensure that the primary shard and its replica shards are allocated in a balanced manner across available, distinct zones. So maximum shard copies per zone would equal `ceil (number_of_shard_copies/number_of_distinct_zones)`.

Reviewer: ```suggestion
You can set custom node attributes on OpenSearch nodes to be used for shard allocation awareness. For example, you can set the `zone` attribute on each node to represent the zone in which the node is located. You can also use the `zone` attribute to ensure that the primary shard and its replica shards are allocated in a balanced manner across available, distinct zones. In this scenario, maximum shard copies per zone would equal `ceil (number_of_shard_copies/number_of_distinct_zones)`.
```

> OpenSearch, by default, allocates shard copies of a single shard across different nodes. When only 1 zone is available, such as after a zone failure, OpenSearch allocates replica shards to the only remaining zone i.e. it considers only available zones (attribute values) for calculating maximum allowed shard copies per zone.

Reviewer 2: ```suggestion
OpenSearch, by default, allocates shard copies of a single shard across different nodes. When only 1 zone is available, such as after a zone failure, OpenSearch allocates replica shards to the only remaining zone---it considers only available zones (attribute values) when calculating the maximum number of allowed shard copies per zone.
```

For example, if your index has a total of 5 shard copies (1 primary and 4 replicas) and nodes in 3 distinct zones, then OpenSearch will perform the following to allocate all 5 shard copies:

> - Allocate not more than 2 shards per zone, which will require at least 2 nodes in 2 zones.

Reviewer 2: ```suggestion
- Allocate no more than 2 shards per zone, which will require at least 2 nodes in 2 zones.
```
